aikau/src/main/resources/alfresco/forms/controls/PushButtons.js

```diff
@@ -144,6 +155,30 @@ define(["alfresco/core/CoreWidgetProcessing",
             name: "alfresco/forms/controls/PushButtonsControl",
             config: config
          });
+      },
+
+      /**
+       * Extends the [inherited function]{@link module:alfresco/forms/controls/BaseFormControl#setOptions}
+       * to make use of any configured [noButtonsLabel]{@link module:alfresco/forms/controls/PushButtons#noButtonsLabel}
+       * when an empty options array is provided.
+       * 
+       * @instance
+       * @param {object[]} options An array of the options to be added.
+       * @since 1.0.80
+       */
+      setOptions: function alfresco_forms_controls_BaseFormControl__setOptions(options) {
+         this.inherited(arguments);
+
+         if (options && 
+             options.length === 0 && 
+             this.noButtonsLabel && 
+             this.wrappedWidget)
+         {
+            var span = domConstruct.create("span", {
+               "class": "alfresco-forms-controls-PushButtons__noOptionsLabel"
```

Comment:
You've probably already verified this works, but shouldn't this be `className`? Maybe Dojo handles it more intelligently than pure JS!